asdf/block.py

```diff
@@ -1224,16 +1223,6 @@ def data(self):
         return self._data
 
     def close(self):
-        if self._memmapped and self._data is not None:
-            if NUMPY_LT_1_7:  # pragma: no cover
-                try:
-                    self._data.flush()
-                except ValueError:
-                    pass
-            else:
-                self._data.flush()
-            if self._data._mmap is not None:
-                self._data._mmap.close()
```

Comment:
Nice, we should try to find more of these bits left over from supporting old versions of dependencies and remove them.